sorts/quick_sort_3_partition.py

```diff
@@ -1,4 +1,4 @@
-def quick_sort_3partition(sorting: list, left: int, right: int) -> list:
+def quick_sort_3partition(sorting: list, left: int, right: int):
```

Comment:
Why this change?

Reply:
`quick_sort_3partition` does not return a new list, it just updates in place. So I remove `-> list`.